salt/backups/templates/restore_mongodb.sh

```diff
@@ -2,12 +2,13 @@
 set -e
 
 {% set backupdir = '/backups/{}'.format(settings.get('directory', 'mongodb')) %}
+{% set cachedir = '/backups/.cache/{}'.format(settings.get('directory', 'mongodb')) %}
 mkdir -p {{ backupdir }}
 
 PASSPHRASE={{ settings.duplicity_passphrase }} /usr/bin/duplicity restore\
           --s3-use-server-side-encryption s3+http://odl-operations-backups/{{ settings.get('directory', 'mongodb') }}/ \
-          --archive-dir /backups/.cache \
-          --force --tempdir /backups {{ backupdir }}/
+          --archive-dir {{ cachedir }}
+          --force --tempdir /backups/tmp/ {{ backupdir }}
```

Comment:
Added file.directory to create the /backups dir after the backup drive is mounted